[2024-09-12T02:57:40.473+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-12T02:57:40.493+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: f1_data_pipeline.extract_data manual__2024-09-12T02:57:38.606500+00:00 [queued]>
[2024-09-12T02:57:40.502+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: f1_data_pipeline.extract_data manual__2024-09-12T02:57:38.606500+00:00 [queued]>
[2024-09-12T02:57:40.503+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-09-12T02:57:40.516+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): extract_data> on 2024-09-12 02:57:38.606500+00:00
[2024-09-12T02:57:40.524+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=77) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-12T02:57:40.525+0000] {standard_task_runner.py:72} INFO - Started process 79 to run task
[2024-09-12T02:57:40.526+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'f1_data_pipeline', 'extract_data', 'manual__2024-09-12T02:57:38.606500+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/f1_to_gcs.py', '--cfg-path', '/tmp/tmp64jf62_u']
[2024-09-12T02:57:40.528+0000] {standard_task_runner.py:105} INFO - Job 30: Subtask extract_data
[2024-09-12T02:57:40.573+0000] {task_command.py:467} INFO - Running <TaskInstance: f1_data_pipeline.extract_data manual__2024-09-12T02:57:38.606500+00:00 [running]> on host ac22af181946
[2024-09-12T02:57:40.657+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='f1_data_pipeline' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2024-09-12T02:57:38.606500+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-12T02:57:38.606500+00:00'
[2024-09-12T02:57:40.658+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-12T02:58:08.550+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-12T02:58:08.562+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-12T02:58:08.562+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=f1_data_pipeline, task_id=extract_data, run_id=manual__2024-09-12T02:57:38.606500+00:00, execution_date=20240912T025738, start_date=20240912T025740, end_date=20240912T025808
[2024-09-12T02:58:08.606+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-12T02:58:08.627+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-12T02:58:08.629+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
